components/Navbar.vue

```diff
@@ -8,11 +8,11 @@
     }"
   >
     <div
-      class="container items-center max-lg:!px-0"
-      :class="{ 'is-fluid': !isTouch }"
+      class="grow relative mx-auto my-0 max-lg:!px-0 items-stretch flex min-h-[3.25rem] w-full"
```

Comment:
Can't we reuse this ?

is the same as gallery-item-layout.vue

Reply:
feel free to handle it

Reply:
what's the difference between this and container-fluid?